packages/api/AlvTime.Business/AccessTokens/AccessTokenService.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using AlvTime.Business.Interfaces;
+using AlvTime.Business.Models;
+
+namespace AlvTime.Business.AccessTokens
+{
+    public class AccessTokenService
+    {
+        private readonly IUserContext _userContext;
+        private readonly IAccessTokenStorage _tokenStorage;
+
+        public AccessTokenService(IAccessTokenStorage tokenStorage, IUserContext userContextContext)
+        {
+            _tokenStorage = tokenStorage;
+            _userContext = userContextContext;
+        }
+
+        public AccessTokenDto CreateLifeTimeToken(string friendlyName)
+        {
+            var user = new GenericUser(_userContext.Name, _userContext.Email);
```

Comment:
What is a generic user in this context? Would it be more clean to create a sense of a user in the business, that consists of a name and an e-mail, then let IUserContext have a method like `public User GetCurrentUser()`? Then it would look like:

```
public AccessTokenDto CreateLifeTimeToken(string friendlyName)
{
    var currentUser = _userContext.GetCurrentUser();
    return _tokenStorage.CreateLifeToken(friendlyName, currentUser);
}
```